packages/twenty-server/src/engine/core-modules/auth/dto/exchange-auth-code.input.ts

```diff
@@ -0,0 +1,10 @@
+import { ArgsType, Field } from '@nestjs/graphql';
+
+@ArgsType()
+export class ExchangeAuthCodeInput {
+  @Field(() => String)
+  authorizationCode: string;
+
```

Comment:
Could you also support non PKCE / traditional auth as part of this or maybe do it in another PR but at least in this one introduce the auth type (I'm not sure where that lives? On the app there would be the corresponding auth type maybe?  codeVerifier being only for PKCE

Reply:
Best to do it separate PR?